### PR TITLE
[HIPIFY] Lintian errors: fix package description

### PR DESCRIPTION
**Pull request by david-salinas**
Update package maintainer info, and ensure built binaries are stripped of debug strings.

### CODE REVIEW

File: packaging/hipify-clang.txt [hunk 1]
```diff
@@ -12,9 +12,9 @@ set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
 set(CPACK_PACKAGE_VERSION_MAJOR @LLVM_VERSION_MAJOR@)
 set(CPACK_PACKAGE_VERSION_MINOR @LLVM_VERSION_MINOR@)
 set(CPACK_PACKAGE_VERSION_PATCH @LLVM_VERSION_PATCH@)
-#TODO : Change mail id to standard <package.suppport@amd.com>
-set(CPACK_PACKAGE_CONTACT "Mankov Evgeny <Evgeniy.Mankov@amd.com>")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "hipify-clang: a clang-based tool to translate CUDA source code into portable HIP C++ automatically")
+set(CPACK_PACKAGE_CONTACT "ROCm Compiler Support <rocm.compiler.support@amd.com>")
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Hipify CUDA source.")
+set(CPACK_PACKAGE_DESCRIPTION "This tool translates CUDA source code into portable HIP C++, automatically")
```
Comment (emankov) [inline packaging/hipify-clang.txt:17]: I'd suggest the following:
"The tool translates CUDA source code into portable HIP C++ automatically"

File: packaging/hipify-clang.txt [hunk 2]
```diff
@@ -12,9 +12,9 @@ set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
 set(CPACK_PACKAGE_VERSION_MAJOR @LLVM_VERSION_MAJOR@)
 set(CPACK_PACKAGE_VERSION_MINOR @LLVM_VERSION_MINOR@)
 set(CPACK_PACKAGE_VERSION_PATCH @LLVM_VERSION_PATCH@)
-#TODO : Change mail id to standard <package.suppport@amd.com>
-set(CPACK_PACKAGE_CONTACT "Mankov Evgeny <Evgeniy.Mankov@amd.com>")
-set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "hipify-clang: a clang-based tool to translate CUDA source code into portable HIP C++ automatically")
+set(CPACK_PACKAGE_CONTACT "ROCm Compiler Support <rocm.compiler.support@amd.com>")
```
Comment (emankov) [inline packaging/hipify-clang.txt:15]: Is `rocm.compiler.support@amd.com` a valid e-mail?

File: packaging/hipify-clang.txt [hunk 3]
```diff
@@ -12,9 +12,9 @@ set(CPACK_PACKAGE_VENDOR "Advanced Micro Devices, Inc.")
...
+set(CPACK_PACKAGE_DESCRIPTION_SUMMARY "Hipify CUDA source.")
```
Comment (emankov) [inline packaging/hipify-clang.txt:16]: Remove the period, please.

**Review by emankov — CHANGES_REQUESTED** [review]

File: packaging/hipify-clang.txt (hunk 1 above)
Comment (david-salinas) [reply packaging/hipify-clang.txt:17]: ok

File: packaging/hipify-clang.txt (hunk 2 above)
Comment (david-salinas) [reply packaging/hipify-clang.txt:15]: It is.  Mark created it. We're using it for all ROCm compiler components.

**Review by emankov — APPROVED** [review]

**Pull request merged**